ui/assets/css/_header.scss

```diff
@@ -0,0 +1,19 @@
+.usa-disclaimer {
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+  font-size: 12px;
```

Comment:
Relatedly -- does it make sense to pull these out into more generic mixins? I assume we'll want to reuse this font config elsewhere.